mobify/progressive-web-scaffold
# Adds triggering page views by default to the scaffold using the latest analytics lib from the SDK

Adds triggering page views by default to the scaffold using the latest analytics lib from the SDK

 **JIRA**: https://mobify.atlassian.net/browse/WEB-723
 **Linked PRs**: https://github.com/mobify/progressive-web-sdk/pull/195

## Changes
- Triggers page views when moving between new pages

## How to test-drive this PR
- Clone the progressive-web-sdk and checkout the `WEB-723-add-analytics` branch
- Run `npm link` in progressive-web-sdk and `npm link progressive-web-sdk` in the scaffold
- Navigate to different pages, ensure that the page view analytics requests look correct (I recommend using the GA chrome extension to ensure that's the case)
- Clicked on the themed component button to ensure events are firing correctly
